oak/server/oak_node.h

```diff
@@ -32,7 +32,7 @@ namespace oak {
 
 typedef std::unordered_map<Handle, std::unique_ptr<ChannelHalf>> ChannelHalfTable;
 
-class OakNode final : public Application::Service {
+class OakNode final {
```

Comment:
At some point we should also rename this to WasmNode or something like that?

Reply:
We probably need a generic Node class and a WasmNode, GrpcNode, etc.

Reply:
Yes this would be my preferred terminology, rather than node and pseudo-node.